### ISSUE TITLE
Validation pixAcc: 1.000, mIoU: 0.500

### ISSUE THREAD

**Issue by wangxin1233** [Tramac/awesome-semantic-segmentation-pytorch#26]
![image](https://user-images.githubusercontent.com/44840625/59144471-37fb4c00-8a0b-11e9-9bf3-2ffd4b9bebd2.png)
hi,thank you for your code,now I want to train my own data by pspnet。 this is my first time 
 to use pythorch, and I have added my own data.py  in dataloader file. I don't know why pa  is always 1.00 and miou is 0.5. Could you give me a hand?  thank you very much.

**Comment by Tramac** [comment]
Maybe due to the dataset or the dataloader.

**Comment by wangxin1233** [comment]
my data has two classer :car and background,and my lable's pix is 255.I change /core/data/dataloader/pascal_voc.py as my car.py. I add def get_psp_resnet50_car(**kwargs):
    return get_psp('car', 'resnet50', **kwargs) in pspnet. no error occour but the the loss is not right.could you tell me what should I do next? thank you

**Comment by Tramac** [comment]
The label value shoud be set to 0 and 1.

**Comment by wangxin1233** [comment]
I haven't solved the problem.could you tell you  if I want to train my own data,what should I do ?

**Comment by Tramac** [comment]
You should ensure that the value is 1 for car and 0 for background in label.

**Comment by wangxin1233** [comment]
Could you tell me where should I check my labels? thank you very much

**Comment by Tramac** [comment]
Would u mind showing me your image sample and dataloader script？

**Comment by wangxin1233** [comment]
no problem!
I have solved the problem! 
![image](https://user-images.githubusercontent.com/44840625/59167562-5b85da00-8b64-11e9-960c-13b874bf6782.png)
you are right, the problem is my label. I have 2 classes,but my labels are more than two classes! 
thank you  very much!

**Comment by Tramac** [comment]
Great!
